chapters/synchronous.tex

```diff
@@ -608,12 +612,14 @@ \subsection{Base-clock conversion operators}\doublelabel{base-clock-conversion-o
 
 \begin{longtable}[]{|l|p{12cm}|}
 \hline \endhead
-\textbf{sample}(u, c) &
+\textbf{sample}(u, clock) &
 Input argument u is a continuous-time expression according to
-\autoref{continuous-time-expressions}. The optional input argument c is of type Clock. The operator
-returns a clocked variable that has c as associated clock and has the
-value of the left limit of u when c is active (that is the value of u
-just before the event of c is triggered). If argument c is not provided,
+\autoref{continuous-time-expressions}. The optional input argument clock is of type Clock, and can in a call be given as a named argument (with the name clock),
```

Comment:
There are some really strange line breaks here

Reply:
True, but it works and can clean up later.